### PR TITLE
add wins app

### PR DESCRIPTION
**Pull request by EggsBlue**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/w/wins.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  app "Wins.app"
+  zap trash: [
+    "~/Library/Application Support/cools.wins.main",
+    "~/Library/Application Support/Wins",
+    "~/Library/Caches/cools.wins.main",
+    "~/Library/Caches/cools.wins.main",
+    "~/Library/Containers/WinsHelper",
+    "~/Library/HTTPStorages/cools.wins.main",
+    "~/Library/Preferences/com.tinynudge.pomello.plist",
+  ]
```
Comment (Aaron-212) [inline Casks/w/wins.rb:19]:
```suggestion
url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-1.6.6.dmg",
      verified: "backblazeb2.com/file/winsWebsite/"
  name "Wins"
  desc "Brand new window manager"
  homepage "https://wins.cool/"

  auto_updates true
  depends_on macos: ">= :high_sierra"

  app "Wins.app"

  zap trash: [
    "~/Library/Application Support/cools.wins.main",
    "~/Library/Application Support/Wins",
    "~/Library/Caches/cools.wins.main",
    "~/Library/Containers/WinsHelper",
    "~/Library/HTTPStorages/cools.wins.main",
    "~/Library/Preferences/com.win.cool.plist",
    "~/Library/PreferencesPanes/Wins.prefPane",
  ]
```
Add auto_updates, fix zap stanza and apply `brew style --fix wins`

File: Casks/w/wins.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+cask "wins" do
+  version :latest
+  sha256 :no_check
+
+  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-1.6.6.dmg", verified: "backblazeb2.com/file/winsWebsite/"
+  name "Wins"
+  desc "Brand new window manager"
```
Comment (bevanjkay) [inline Casks/w/wins.rb:7]:
```suggestion
  desc "Window manager"
```

File: Casks/w/wins.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+cask "wins" do
+  version :latest
```
Comment (krehel) [inline Casks/w/wins.rb:2]:
```suggestion
  version "1.6.6"
```

File: Casks/w/wins.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 :no_check
+
+  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-1.6.6.dmg", verified: "backblazeb2.com/file/winsWebsite/"
```
Comment (krehel) [inline Casks/w/wins.rb:5]:
```suggestion
  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-#{version}.dmg", verified: "backblazeb2.com/file/winsWebsite/"
```

File: Casks/w/wins.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+cask "wins" do
+  version :latest
+  sha256 :no_check
```
Comment (krehel) [inline Casks/w/wins.rb:3]: Replace with the actual value.
```suggestion
  sha256 :no_check
```

**Comment by EggsBlue** [comment]
added auto-updates and passed brew style

**Comment by EggsBlue** [comment]
image

I think I passed the style check, but I don’t know why CI failed

**Comment by Aaron-212** [comment]
> image
> 
> I think I passed the style check, but I don’t know why CI failed

Did you commit and push the changes after running `brew style --fix <cask>`?

**Comment by EggsBlue** [comment]
sorry, my bad. 

looks like I forget commit the correct ones

take a look this one

**Comment by krehel** [comment]
As best practice, this should be its own branch.  Don't work on `master`.

And, there are issues because merge commits are happening on the branch., which is going to require rebasing.  Which is another reason why not to work on `master`

**Pull request closed without merging**